src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -32,6 +35,11 @@ def setUpClass(cls):
         cls.user1 = 'foo@foo.com'
         cls.id1 = 'subscriptions/1'
         cls.display_name1 = 'foo account'
+        cls.home_account_id = "00000003-0000-0000-0000-000000000000@00000003-0000-0000-0000-000000000000"
```

Comment:
The separator in `home_account_id` is a dot: `182c8534-f413-487c-91a3-7addc80e35d5.54826b22-38d6-4fb2-bad9-b7b93a3e9c5a`.

Just FYI, there is an ongoing discussion about whether Azure CLI and Azure Identity need to know about `home_account_id` which is internal to MSAL.